### DISCUSSION TITLE
How can I add a user's phone number and send a confirmation message if the user has registered with their email?

### DISCUSSION THREAD

**Discussion by RafoPetrosyan** [supabase/supabase#15839] [Questions]
How can I add a user's phone number and send a confirmation message if the user has registered with their email?

![image](https://github.com/supabase/supabase/assets/94311876/da703850-b683-4938-bd91-b520ff696ad1)

**Comment by buff** [comment]
Hi @RafoPetrosyan, did you make any progress on this... I have a similar use case??
